### PR TITLE
Transform changes

### PR DESCRIPTION
**Pull request by Djelibeybi**
This enables a transform to transition the color along with the brightness if the power is turned on at the beginning of the transform. Useful for transitioning from a night light to bright white.

Also fixed a typo I noticed along the way, at no extra charge. :)

Requires https://github.com/delfick/photons-core/pull/8

### CODE REVIEW

**Comment by Djelibeybi** [comment]
Travis is failing because it's pulling in the stable version of Photons Core, not the updated version.

File: photons_interactor/commander/commands/control.py [hunk 1]
```diff
@@ -94,7 +93,7 @@ class TransformCommand(store.Command):
 
     async def execute(self):
         fltr = chp.filter_from_matcher(self.matcher, self.refresh)
-        msg = Transformer.using(self.transform)
+        msg = Transformer.using(self.transform, **self.transform_options)
```
Comment (delfick) [inline photons_interactor/commander/commands/control.py:114]: you still need to define this field with a dictobj.Field

Comment (Djelibeybi) [inline photons_interactor/commander/commands/control.py:114]: This generates the following error for me :`AttributeError: 'TransformCommand' object has no attribute 'transform_options'`. I thought I'd updated photons-core in the venv properly, but I pushed it anway, so you can take a look too.

**Comment by delfick** [comment]
OK, master is now using 0.24.7 with the transform changes, please rebase this :)

Also, feel free to add to https://github.com/delfick/photons-interactor/blob/master/docs/usage/changelog.rst

**Comment by Djelibeybi** [comment]
Rebased (badly, I'll admit) and changelog updated.

**Comment by delfick** [comment]
Try

```
git fetch origin
git reset origin/master
git commit -av
git push origin transform -f
```

**Comment by Djelibeybi** [comment]
One day I will do git things without StackOverflow (or similar). Today was not that day.

**Comment by delfick** [comment]
When you're ready for that, I highly recommend https://git-scm.com/book/en/v2

**Comment by delfick** [comment]
I'll release this new version a bit later. Depends if I find time to make the effects commands as well or not

**Comment by Djelibeybi** [comment]
> Depends if I find time to make the effects commands as well or not

I started playing with this, then fell down the rabbit hole of creating `multizone/get`, `multizone/set` and `multizone/effect` and then started to get dirty looks from my husband as I was already 1.5 hours late for breakfast. :)

**Comment by delfick** [comment]
hahhahah.

I'd make a 

firmware_effects/start
firmware_effects/stop
firmware_effects/status

**Comment by Djelibeybi** [comment]
Yeah, that works too. :)

**Pull request merged**